### PR TITLE
GitHub Balance Points

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR adds the GitHub balance points (GBP from herein) system to the repo. This will automatically track user point values from pull requests based on labels and is a way of automating "Fix tokens" as per the feature freeze in #17040 

@Fox-McCloud 

## Why It's Good For The Repo
Automating this is good. It also means we can have values for other labels other than `Fix`

### CODE REVIEW

File: .github/gbp.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+"Code Improvement" = 1
+"Feature" = -9
+"Fix" = 3
+"GC Related" = 12
+"Grammar and Formatting" = 1
+"High Priority" = 20
+"Infrastructure" = 5
+"Performance" = 12
+"Refactor" = 3
+"Sound" = 3
```
Comment (hal9000PR) [inline .github/gbp.toml:16]: sound seems a bit high in my opinion. being on the same level as a fix/refactor.

**Review by hal9000PR — COMMENTED** [review]
just my opinions. good idea overall.

File: .github/gbp.toml (hunk 1 above)
Comment (AffectedArc07) [reply .github/gbp.toml:16]: Could probably make it 1

**Comment by Sinestia** [comment]
i thought it was called goodboy points

have i been wrong this whole time

**Comment by AffectedArc07** [comment]
> i thought it was called goodboy points
> 
> have i been wrong this whole time

Yes

**Review by variableundefined — APPROVED** [review]

**Comment by S34NW** [comment]
How many points for some tendies?

**Pull request merged**